book/doctrine.rst

```diff
@@ -916,8 +924,81 @@ a ``name`` field and the associated getter and setter functions.
 Relationship Mapping Metadata
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
I really like that you switched the order to show ManyToOne first. I wrote this original spot, but it's bothered me for a long time that we showed OneToMany first (I hadn't had a chance to change it).